packages/integration-tests/tests/retrospective-demo/retrospective-demo-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import {Page, expect, Locator} from '@playwright/test'
+import config from '../config'
+
+export async function startDemo(page: Page) {
+  await config.goto(page, '/retrospective-demo')
+  await page.click('text=Start Demo')
+}
+
+export async function goToNextPhase(page: Page) {
+  const nextButton = page.locator('button :text("Next")')
+  await expect(nextButton).toBeVisible()
+
+  // You "confirm" going to the next phase by clicking the next button twice
+  await nextButton.click()
+  await nextButton.click()
+}
+
+export async function skipToGroupPhase(page: Page) {
+  await startDemo(page)
+  await goToNextPhase(page)
+  expect(page.url()).toEqual(`${config.rootUrlPath}/retrospective-demo/group`)
+}
+
+export async function skipToVotePhase(page: Page) {
+  await skipToGroupPhase(page)
+  await goToNextPhase(page)
+  expect(page.url()).toEqual(`${config.rootUrlPath}/retrospective-demo/vote`)
+}
+
+export async function skipToDiscussPhase(page: Page) {
+  await skipToVotePhase(page)
+  await goToNextPhase(page)
+  expect(page.url()).toEqual(`${config.rootUrlPath}/retrospective-demo/discuss/1`)
+}
+
+export async function dragReflectionCard(
```

Comment:
I tried to improve this method to support touch events for moving cards on mobile between screens.

For historical sake, here's what that looked like:

```ts
export async function dragReflectionCard(
  page: Page,
  cardToDrag: Locator,
  destination: Locator,
  options: {isMobile: boolean; viewport: ViewportSize; waitForAnimation?: boolean}
) {
  const {isMobile, viewport, waitForAnimation = true} = options

  if (isMobile) {
    await cardToDrag.scrollIntoViewIfNeeded()
    await (await cardToDrag.elementHandle()).waitForElementState('stable')
    await (await cardToDrag.elementHandle()).waitForElementState('enabled')

    const cardToDragBox = await cardToDrag.boundingBox()
    await page.mouse.move(cardToDragBox.x, cardToDragBox.y)
    await new Promise((r) => setTimeout(r, 1_000))
    await page.mouse.down()
    await new Promise((r) => setTimeout(r, 1_000))
    await page.mouse.move(viewport.height * 0.1, viewport.width / 2)
    await new Promise((r) => setTimeout(r, 10_000))
    const destinationBox = await destination.boundingBox()
    await page.mouse.move(
      destinationBox.x + destinationBox.width / 2,
      destinationBox.y + destinationBox.height / 2
    )
    await page.mouse.up()
  } else {
    await cardToDrag.dragTo(destination)
  }
  // await cardToDrag.dragTo(destination)

  if (waitForAnimation) {
    // When dragging the card from one place to another, a "clone" is created
    // during the animation. Therefore, wait for the card animation to complete
    // to make further assertions.
    await expect(cardToDrag).toHaveCount(1)
  }
}
```

However, the logic to switch the "pane" on mobile relies on `touch` HTML events (not `click`, like I'm simulating here. There's an open Playwright issue that would help make this easier (please upvote if you've got a sec: https://github.com/microsoft/playwright/issues/2903).

For now, I test drag-and-drop grouping in the same pane/column. For the test where I drag between columns, I skip mobile devices.